python/tests/test_account.py

```diff
@@ -51,9 +51,30 @@ def test_chat(self, acfactory):
 
         chat2 = ac1.create_chat_by_contact(contact1.id)
         assert chat2.id == chat.id
+        assert chat2.get_name() == chat.get_name()
         assert chat == chat2
         assert not (chat != chat2)
 
+        for ichat in ac1.get_chats():
+            if ichat.id == chat.id:
+                break
+        else:
+            pytest.fail("could not find chat")
+
+    def test_group_chat_creation(self, acfactory):
+        ac1 = acfactory.get_offline_account()
+        contact1 = ac1.create_contact("some1@hello.com", name="some1")
+        contact2 = ac1.create_contact("some2@hello.com", name="some2")
+        chat = ac1.create_group_chat(name="title1")
+        chat.add_contact(contact1)
+        chat.add_contact(contact2)
+        assert chat.get_name() == "title1"
+        assert contact1 in chat.get_contacts()
```

Comment:
not sure was the in really does and no concrete idea about how python objects work, it may just be that we're comparing different objects here where we want to compare the ids.

eg. would `contact1 = ac1.create_contact("same@same.same")` and `contact2 = ac1.create_contact("same@same.same")` return the _same_ objects? the ids returned by `dc_create_contact()` will be the same.

so `contact1.id == contact2.id` will work, not sure if `contact1 == contact2` is true or false.

Reply:
don't worry -- "contact1 == contact2" implies "contact1.id == contact2.id and contact1._dc_context == contact2._dc_context.